### PR TITLE
Org action fixes and form cleanup

### PR DESCRIPTION
**Pull request by jolheiser**

### CODE REVIEW

**Review by 6543 — APPROVED** [review]

**Review by zeripath — APPROVED** [review]
I'd probably prefer if the uids weren't in the query but it's fine.

**Comment by 6543** [comment]
I can send a patch on its own if this is merged ...?

File: templates/org/member/members.tmpl [hunk 1]
```diff
@@ -54,9 +54,15 @@
 					<div class="ui four wide column">
 						<div class="text right">
 							{{if eq $.SignedUser.ID .ID}}
-								<a class="ui red small button" href="{{$.OrgLink}}/members/action/leave?uid={{.ID}}">{{$.i18n.Tr "org.members.leave"}}</a>
+								<form method="post" action="{{$.OrgLink}}/members/action/leave?uid={{.ID}}">
```
Comment (6543) [inline templates/org/member/members.tmpl]:
```suggestion
								<form method="post" action="{{$.OrgLink}}/members/action/leave">
```

File: templates/user/settings/organization.tmpl [hunk 2]
```diff
@@ -17,7 +17,10 @@
 					{{range .Orgs}}
 					<div class="item">
 						<div class="right floated content">
-							<a class="ui blue small button" href="{{AppSubUrl}}/org/{{.Name}}/members/action/leave?uid={{.ID}}">{{$.i18n.Tr "org.members.leave"}}</a>
+							<form method="post" action="{{AppSubUrl}}/org/{{.Name}}/members/action/leave?uid={{.ID}}">
```
Comment (6543) [inline templates/user/settings/organization.tmpl]:
```suggestion
							<form method="post" action="{{AppSubUrl}}/org/{{.Name}}/members/action/leave">
```

**Review by 6543 — CHANGES_REQUESTED** [review]
leave action don't need `?uid={{.ID}}`

**Comment by jolheiser** [comment]
No, but if `uid` is `0` it will redirect.

**Review by 6543 — APPROVED** [review]

**Comment by jolheiser** [comment]
I have a PR ready after this is merged for cleaning up the forms. 👍

**Comment by jolheiser** [comment]
Okay, PR now has form fixes, please re-review.

**Review by zeripath — APPROVED** [review]

**Review by guillep2k — APPROVED** [review]

**Review by 6543 — COMMENTED** [review]
lgtm :)

**Pull request merged**